### DISCUSSION TITLE
guide/waypoint

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#905] [Announcements]
# guide/waypoint

路径点 提示 打开ClickGui后可以点击左下角第二个按钮打开路径点快捷编辑页面 /img.png 从左到右依次为 路径点名字 所属世界 删除 编辑 开关可视状态 /img_1.png 命令 快捷键

https://docs.seiua.moe/guide/waypoint.html

**Comment by seiuna** [comment]
123
